src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Circle extends FigureAbstract implements Figure {
```

Comment:
What's the point of having FigureAbstract class and then a Figure interface? 🤔 Separate interfaces to Drawable, AreaCalculator, make abstract class Figure implement them, and you won't need to write implements explicitly for each figure

Reply:
when i use mvn clean package a get 29 problems with checklist ,but mvn don't show where is the problem, i'm reformat my code using default mac tool, and nothing changes, how i can figure out where is the checklist problems?